.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+## Before you get started
+- [ ] 🙋‍♀️ Create an issue to discuss what you are doing to do!
+
+---
+
+## Description
+- Closes #issue_number
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. 
+
+## Type of change
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
```

Comment:
Maybe name it shorter, e.g. "Documentation change"? 
This would apply for issues which only change code documentation, README etc. 

All other issues, normally require also updates of documentation! See checklist below.

Reply:
Done!